docs/entities/servicecatalog.md

```diff
@@ -30,5 +30,12 @@ The Catalog will also be removed from the matching application namepsaces.
 
 ### Update
 
-When a registered service is being updated the Service Catalog will get updated too.
-The updated Sevice Catalog will be made available to all the application namespaces of matching Cluster Environment by Primaza.
+When a registered service is being updated, the appropriate service catalog will get updated too.
+The updated service catalog will be made available to all the application namespaces of matching cluster Eenvironment by Primaza.
+The constraint section of a registered service defines a list of environments.
+The environment list can contain explicit environments that are to include the service or it can also contain a list of environments that are to exclude the service.
+The way to depict exclusion is by adding the ! prefix to the environment, i.e !prod.
+Exclusion has precedence over inclusion for example, if the list contains !prod but also includes dev, then dev is considered to be in the !prod set of environments and therefore redundant.
```

Comment:
Line 38 should probably come with a `Please Note`

Reply:
OK